plugins/module_utils/apitoken.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+import traceback
+
+try:
+    import requests
+except ImportError:
+    HAS_REQUESTS = False
+    REQUESTS_IMPORT_ERROR = traceback.format_exc()
+else:
+    HAS_REQUESTS = True
+    REQUESTS_IMPORT_ERROR = None
+
+URL = "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token"
+
+
+def _get_refresh_token(offline_token):
+    # Set headers
+    headers = {
+        'Content-Type': 'application/x-www-form-urlencoded',
+        'Accept': 'application/json'
+    }
+    # set data
+    data = {}
+    data['grant_type'] = "refresh_token"
+    data['client_id'] = "cloud-services"
+    data['refresh_token'] = offline_token
+
+    # Generate API token
+    response = requests.post(f"{URL}", data=data, json=None, headers=headers)
```

Comment:
Don´t pass json if not needed

```suggestion
    response = requests.post(f"{URL}", data=data, headers=headers)
```